### PR TITLE
added some conversions

### PR DESCRIPTION
**Pull request by veds12**
* Fixed typo in commander
* Added ```traj_to_msg``` conversion
* Added ```msg_to_traj``` conversion

**To Do** : 

* Add ```Odom_to_RobotState``` conversion
* Add ```Velocity_to_Twist``` conversion

### CODE REVIEW

**Comment by veds12** [comment]
@threewisemonkeys-as I think these conversions should suffice for now. Lmk if there are any else that need to be added

File: src/gennav_ros/conversions.py [hunk 1]
```diff
@@ -97,3 +97,72 @@ def msg_to_traj(msg):
 
     traj = Trajectory(path=path, timestamps=timestamps)
     return traj
+
+
+def Odom_to_RobotState(msg):
```
Comment (threewisemonkeys-as) [inline src/gennav_ros/conversions.py:102]: can you keep this lowercase

File: src/gennav_ros/conversions.py [hunk 2]
```diff
@@ -97,3 +97,72 @@ def msg_to_traj(msg):
...
+    velocity = Velocity(linear=linear_vel, angular=angular_vel)
+
+    state = RobotState(
+        position=position, orientation=orientation, velocity=velocity
+    )
+
+    return state
+
+
+def Velocity_to_Twist(velocity):
```
Comment (threewisemonkeys-as) [inline src/gennav_ros/conversions.py:149]: same here

**Comment by threewisemonkeys-as** [comment]
Could you also add the inverse transformations? `robotstate_to_odom` and `twist_to_velocity`

**Comment by veds12** [comment]
Where would they be needed?

**Comment by threewisemonkeys-as** [comment]
hmm yeah 😅

No need then

**Pull request merged**